tests/unit/base/trained_model_serialization_unittest.cc.py

```diff
@@ -6,20 +6,30 @@
 # (at your option) any later version.
 #
 
-# Classes that need custom initialization
-IGNORE_MACHINES = {
-    'CLinearMachine' : [
-        'CFeatureBlockLogisticRegression', 'CLibLinearMTL',
-        'CMultitaskLinearMachine', 'CMultitaskLogisticRegression',
-        'CMultitaskL12LogisticRegression', 'CMultitaskLeastSquaresRegression',
-        'CMultitaskTraceLogisticRegression', 'CMultitaskClusteredLogisticRegression',
-        'CLatentSVM', 'CLatentSOSVM', 'CDomainAdaptationSVMLinear'
-    ],
-    'CKernelMachine' : [
-        'CDomainAdaptationSVM', 'CMKLRegression',
-        'CMKLClassification', 'CMKLOneClass', 'CSVM'
-    ]
-}
+# Ignore classes that need custom initialization and:
```

Comment:
This comment confuses me. I understand that these classes are ignored.
But I wonder whether they are tested elsewhere, what custom initialization means, and the C<=0 error seems to be a bug that should be referenced with a github issue?